eks-ebs-enable/core/plan/const.go

```diff
@@ -0,0 +1,31 @@
+package plan
+
+const (
+	defaultAudience = "sts.amazonaws.com"
+
+	driverRoleNameFormat = "EBS_CSI_%s"
+
+	ebsPolicyARN = "arn:aws-cn:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy"
```

Comment:
Why `aws-cn` in ARN?

Reply:
Actually this doesn't look like a problem - the role is rendered correctly without any `-cn` prefixes in my case.

Reply:
```suggestion
	ebsPolicyARN = "arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy"
```

Reply:
Changed so partition can be passed in....but it defaults to `aws`